pkgs/development/python-modules/aiosonic/default.nix

```
@@ -0,0 +1,59 @@
+{
+  pkgs,
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchFromGitHub,
+  poetry-core,
+  # install_requires
+  charset-normalizer,
+  h2,
+  onecache,
+  pytest,
+}:
+
+buildPythonPackage rec {
+  pname = "aiosonic";
+  version = "0.20.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+  doCheck = true;
+
+  src = fetchFromGitHub {
+    owner = "sonic182";
+    repo = "aiosonic";
+    rev = "refs/tags/${version}";
+    hash = "sha256-RMkmmXUqzt9Nsx8N+f9Xdbgjt1nd5NuJHs9dzarx8IY=";
+  };
```

```suggestion
  };
  
    postPatch = ''
      substituteInPlace pytest.ini --replace-fail \
        "addopts = --black --cov=aiosonic --cov-report term --cov-report html --doctest-modules" \
        "addopts = --doctest-modules"
  '';
```
this is needed, because we don't need to run coverage or `black`